pkg/util/bandwidth/linux.go

```diff
@@ -128,15 +128,16 @@ func asciiCIDR(cidr string) (string, error) {
 	return fmt.Sprintf("%s/%d", ip.String(), size), nil
 }
 
-func (t *tcShaper) findCIDRClass(cidr string) (class, handle string, found bool, err error) {
+func (t *tcShaper) findCIDRClass(cidr string) (classAndHandle [][]string, found bool, err error) {
+	result := [][]string{}
```

Comment:
could have a more meaningful variable name instead of `result`.

Reply:
What do you suggest about this?